From 4982a10fe525a548db8850c5a759b7ff22ad09ee Mon Sep 17 00:00:00 2001
From: motten <motten@90b65887-3827-0410-9a23-83215b262276>
Date: Thu, 22 Dec 2016 12:18:19 +0000
Subject: [PATCH 0812/1076] MANTIS 0032431 mControl version  1.1.1.0   \li
 13.12.2016   \li adaptation - MOt Mantis 0032431   -adapted tracing levels
 and information for logging-events   -deleted event CMD_MASSAIREBOOT_REMOTE
 and CMD_MASSAISHUTDOWN_REMOTE and sourced    the functionality to the
 mControl::Servant

git-svn-id: svn://localhost/SelfServiceCommon/trunk@1121 90b65887-3827-0410-9a23-83215b262276
---
 .../Interfaces/inc/Tools/Notification.h             | 13 ++++---------
 1 file changed, 4 insertions(+), 9 deletions(-)

diff --git a/SelfServiceCommon/Interfaces/inc/Tools/Notification.h b/SelfServiceCommon/Interfaces/inc/Tools/Notification.h
index 24c3798c..35f84d13 100644
--- a/SelfServiceCommon/Interfaces/inc/Tools/Notification.h
+++ b/SelfServiceCommon/Interfaces/inc/Tools/Notification.h
@@ -32,10 +32,8 @@ namespace MASSAI
   {
     typedef enum
     {
-      //Terminate the thread
       CMD_START_LOGGING = 1000,
 
-      //Initiate the thread specific work
       CMD_STOP_LOGGING,
 
       CMD_LOG_EVENT
@@ -44,14 +42,12 @@ namespace MASSAI
 
     typedef enum
     {
-      //Shutdown mStarter.exe
       CMD_SHUTDOWN_PLATFORM = 2000,
 
     } MC_TERMINATOR_THREAD_MSG;
 
     typedef enum 
     {
-      //Terminate the thread
       CMD_THREAD_SHUTDOWN = 3000,
 
       CMD_THREAD_DO_WORK
@@ -66,11 +62,7 @@ namespace MASSAI
       CMD_MASSAISTART_SCHEDULED,
         
       CMD_MASSAISTOP_REMOTE, 
-      CMD_MASSAISTOP_SCHEDULED,   
-
-      CMD_MASSAIREBOOT_REMOTE,
-
-      CMD_MASSAISHUTDOWN_REMOTE
+      CMD_MASSAISTOP_SCHEDULED
 
     } MCONTROL_MSG;
 
@@ -78,6 +70,9 @@ namespace MASSAI
     class ThreadNotification : public Poco::Notification 
     {
     public:
+        /*
+        * Constructor
+        */
         ThreadNotification<tempMsg>(std::string info, std::string pID, tempMsg msg) :
           m_info(info),
           m_parentID(pID),
-- 
2.41.0.windows.1

